### PR TITLE
feat: remove alcohol stuff & move consumables to server config

### PR DESCRIPTION
**Pull request by TonybynMp4**
## Description
For my awesome [qbx_alcoholism ](https://github.com/tonybynmp4/qbx_alcoholism) resource (we need to use it)

## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: server/consumables.lua [hunk 1]
```diff
@@ -11,6 +11,7 @@ for alcohol, params in pairs(sharedConfig.consumables.alcohol) do
 
         local sustenance = player.PlayerData.metadata.thirst + math.random(params.min, params.max)
         player.Functions.SetMetaData('thirst', sustenance)
+        Player(source).state:set('alcohol', (Player(source).state.alcohol or 0) + params.alcoholLevel, true)
```
Comment (ghost) [inline server/consumables.lua]:
```suggestion
        local playerState = Player(source).state
        playerState:set('alcohol', (playerState.alcohol or 0) + params.alcoholLevel, true)
```
Gotta change this g

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by bicatcho11** [comment]
hi why not using function CheckPlayerData(source, playerData) ?for save alcohol level

**Comment by BerkieBb** [comment]
> hi why not using function CheckPlayerData(source, playerData) ?for save alcohol level

CheckPlayerData is for initializing PlayerData, it's an internal function not meant for anything else

**Comment by TonybynMp4** [comment]
I think i'll repurpose this pr to yeet alcohol from smallresources tbh,
With qbx_alcoholism we can just have the alcohol consumables there

**Review by BerkieBb — APPROVED** [review]

**Comment by TonybynMp4** [comment]
Using qbx_alcoholism isn't planned so i'll close this.

**Pull request closed without merging**